misc/python/materialize/mzcompose/__init__.py

```diff
@@ -79,6 +79,8 @@
     "timestamp_oracle": "postgres",
     "default_idle_arrangement_merge_effort": "0",
     "default_arrangement_exert_proportionality": "16",
+    # TODO: remove this once #18628 is resolved
+    "log_filter": "mz_storage::sink::kafka=DEBUG,info"
```

Comment:
```suggestion
    "log_filter": "mz_storage::sink::kafka=debug,info"
```

not _positive_ this matters, but lowercase def works, not sure about uppercase